### DISCUSSION TITLE
invert consumed am-hours measurement

### DISCUSSION THREAD

**Discussion by hansgeorgk** [Louisvdw/dbus-serialbattery#143] [Q&A] [answered]
Hi,

installed v0.12 - looks great. thanks a lot for your work.
I can now enter the installed capacity and I see the consumed amphours from my Daly BMS :-) 👍 

is it possible to invert consumed amphours measurement?
I would like to have negative number - similar to how the BMV is displaying it.

cheers
HG

**Comment by Louisvdw** [comment, marked as answer]
Hi @hansgeorgk 
There is no option to invert the consumed power at the moment
